packages/scoutgame-ui/src/components/common/NFTPurchaseDialog/components/ChainSelector/ChainSelector.tsx

```diff
@@ -95,12 +98,13 @@ function SelectField(
             t.chainId === _chain.id &&
             (_chain.currency === 'ETH'
               ? t.address === NULL_EVM_ADDRESS
-              : t.address?.toLowerCase() === _chain.usdcAddress.toLowerCase())
+              : t.address?.toLowerCase() === _chain.usdcAddress.toLowerCase() ||
+                t.address?.toLowerCase() === scoutTokenErc20ContractAddress().toLowerCase())
         );
         let _balance = Number(_tokenBalanceInfo?.balance);
 
         if (_balance) {
-          if (_chain.currency === 'ETH') {
+          if (_chain.currency === 'ETH' || _chain.currency === 'SCOUT') {
```

Comment:
should be DEV!

Reply:
updated since we have launched out token